docs/user-guide/global-configurations/tags-policy.md

```diff
@@ -1,51 +1,165 @@
 
-## Add Tags
+The **Tags Policy** feature lets you create tags for applications, and enforce deployment-specific rules if needed. It also allows you to propagate these tags as labels to the Kubernetes resources associated with the application.
 
-To add tags, follow these steps:
+![Figure 1: How Tags Policy Works](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/mandatory-tag.gif)
 
-1. From the left pane, navigate to the `Global Configuration` section.
-2. Select `Tags` within the `Global Configuration` section.
+## Adding Tags
 
-Once you are in the `Tags` section, locate the **Add Tag** button in the upper-right corner of the screen and click it.
+{% hint style="warning" %}
+### Who Can Perform This Action?
+Users need to have super-admin permission to add tags.
+{% endhint %}
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/add-tag.jpg)
+1. Go to **Global Configurations** → **Tags Policy**.
 
-Within the `Add Tag` section, you will find two options for tags:
+    ![Figure 2: Tags Policy](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/gc-tags-policy.jpg)
 
-* **Suggested tags**: These tags appear as suggestions when adding tags to applications.
-* **Mandatory tags**: These tags are required for applications within the selected project.
+2. Click **+ Add Tag**.
 
-To create mandatory tags, choose the second option: `Mandatory tags`. This ensures that the specified tags are mandatory for the applications within the selected project.
+    ![Figure 3: 'Add Tag' Button](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/add-tags-gc.jpg)
 
-3. Next, choose the project(s) for which you want to create mandatory tags. You can select multiple projects at once (if required).
+3. **Suggested tags/Mandatory tags** - If you choose and create `Suggested tags`, they will appear as [dropdown suggestions](#appearance-of-suggested-tags) when adding tags to applications. However, if you create `Mandatory tags` and enforce deployment restrictions, the user must configure the tag for applications within the selected project (refer next step).
 
-4. After selecting the projects, proceed to add the mandatory tags for the selected projects.
+    ![Figure 4: Creating Suggested or Mandatory Tag](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/suggested-mandatory.gif)
 
-![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/global-configurations/mandatory-tags/create-tag.jpg)
+4. **Select Project(s)** - (Not available for `Suggested tags`) Select the projects whose applications will have the mandatory tags enforced. For all other projects, mandatory tags will appear just as a suggestion.
```

Comment:
To mandate a tag, you must provide a project where this policy should apply. suggested tags are suggested globally